finish/src/main/java/io/openliberty/guides/system/SystemResource.java

```diff
@@ -1,6 +1,6 @@
 // tag::copyright[]
 /*******************************************************************************
- * Copyright (c) 2017, 2018 IBM Corporation and others.
+ * Copyright (c) 2017, 2018, 2019 IBM Corporation and others.
```

Comment:
copyright years here should be `2017, 2019`

Reply:
addressed.